index.d.ts

```diff
@@ -25780,10 +25836,6 @@ declare namespace Octokit {
      * The contents of the comment.
      */
     body: string;
-    /**
-     * **Deprecated**. Use **position** parameter instead. Line number in the file to comment on.
-     */
-    line?: number;
```

Comment:
Do you know why this was removed? I don't even understand why `IssuesUpdateCommentParams` would have a `line` parameter in the first place, I'm a little confused.

Reply:
This is another weird diff thing. On master the `IssuesUpdateCommentParams` type definition looks like

```ts
  export type IssuesUpdateCommentParams = {
    /**
     * The contents of the comment.
     */
    body: string;

    comment_id: number;

    owner: string;

    repo: string;
  };
```

The line field should be moved to the `ReposCreateCommitCommentParamsDeprecatedLine`, which it shows up in here https://github.com/octokit/rest.js/pull/1496/files#diff-b52768974e6bc0faccb7d4b75b162c99R25865, but that diff is missing some fields like in the `IssuesUpdateParamsDeprecatedAssignee` type. But in the actual file it looks like everything's there.